### PR TITLE
Display input/output placeholders

### LINKED ISSUE #552: Improve first intuition of understanding the decision table by layout by adding default names
__Is your feature request related to a problem? Please describe.__

Screenshot_20200729_113508
Some new users expressed that they had difficulties of clearly understanding where are outputs and where are inputs based on the decision table layout. 
They expressed two reasons:
* The `When` and `Then` are rather small and not spotted quickly
* The `+` signs give the intuition that they connect the columns using a boolean AND logic (which contradicts the input and output column concept"

__Describe the solution you'd like__

A fist approach to resolve the issue is:
- [ ] Prefill the columns with `Input1` and `Output1`
Screenshot_20200729_113353
- [ ] ~~Move the `+` symbols a bit to the left and top~~

__Describe alternatives you've considered__

n/a

__Additional context__

Collected during DMN training

### PR DESCRIPTION
**Pull request by barmac**
Closes #552

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by barmac** [comment]
Build artifacts:

- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/552-add-column-name-placeholders/camunda-modeler-552-add-column-name-placeholders-linux-x64.tar.gz
- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/552-add-column-name-placeholders/camunda-modeler-552-add-column-name-placeholders-mac.dmg
- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/552-add-column-name-placeholders/camunda-modeler-552-add-column-name-placeholders-mac.zip
- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/552-add-column-name-placeholders/camunda-modeler-552-add-column-name-placeholders-win-ia32.zip
- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/552-add-column-name-placeholders/camunda-modeler-552-add-column-name-placeholders-win-x64.zip

File: packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css [hunk 1]
```diff
@@ -317,6 +317,30 @@
   text-overflow: ellipsis;
 }
 
+.dmn-decision-table-container thead {
+  counter-reset: inputs outputs;
```
Comment (MaxTru) [inline packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css:321]: :+1: how we use `counters`

**Review by MaxTru — APPROVED** [review]
:wave:

File: packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css (hunk 1 above)
Comment (barmac) [reply packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css:321]: I should use that for the When/And/Then clauses too.

**Review by andreasgeier — APPROVED** [review]
The requirement is fulfilled. And it is an improvement compared to the previous placeholder "-".

But still I think we need to observe this topic. I'm afraid that some users may not understand that: (1) it is a placeholder and not a value / (2) double click is needed to change the name / (3) in the overlay there is an _invisible_ input (without placeholder) to add a column title.

**Comment by barmac** [comment]
Iteration \#2 pushed on branch.

**Review by MaxTru — CHANGES_REQUESTED** [review]
The placeholder is only visible in the header when not having the edit context menu open. It should also be visible when double-clicking the header and opening the context menue for in place editing.

**Comment by philippfromme** [comment]
I agree with @MaxTru regarding the missing placeholder in the editor.

**Comment by barmac** [comment]
Ready for another round. The artifacts should be ready in 15 minutes.

File: packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css [hunk 2]
```diff
@@ -317,6 +317,16 @@
   text-overflow: ellipsis;
 }
 
+.dmn-decision-table-container thead .input-expression:empty:before {
+  content: "Input";
```
Comment (pinussilvestrus) [inline packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css:321]: does it have to be double quotes? We use single quotes in other places

**Review by pinussilvestrus — APPROVED** [review]
Besides a minor thing in the css, it seems to work now as expected

Kapture 2020-10-01 at 13 49 07

File: packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css (hunk 2 above)
Comment (philippfromme) [reply packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css:321]: We use double quotes in HTML/JSX and single quotes in all other places.

Comment (barmac) [reply packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css:321]: I'll change that for sake of consistency.

**Pull request merged**